testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/hibernate/SFSBHibernate2LcacheStats.java

```diff
@@ -97,7 +95,7 @@ public void setupConfig() {
             // build the serviceregistry
             sessionFactory = configuration.buildSessionFactory();
         } catch (Throwable ex) { // Make sure you log the exception, as it might be swallowed
-            System.err.println("Initial SessionFactory creation failed." + ex);
+            ex.printStackTrace();
```

Comment:
Same for SFSBHibernateSessionFactory, SFSBHibernateTransaction, SFSBHibernateEnversSessionFactory, SFSBHibernateSFNaturalId, TxTimeoutTestCase (there are "//" instead of logger usage), SFSB classes